py-scripts/lf_interop_video_streaming.py

```diff
@@ -236,22 +248,40 @@ def build(self):
         self.total_duration = ""
         self.formatted_endtime_str = ""
         self.phone_data = self.get_resource_data()
-        
-        self.http_profile.direction = 'dl'
-        self.http_profile.dest = '/dev/null'
-        self.http_profile.max_speed = self.max_speed
-        self.http_profile.requests_per_ten = self.urls_per_tenm
-        upload_name=self.phone_data[-1].split('.')[-1]
+
+        self.direction = 'dl'
+        self.dest = '/dev/null'
+        self.max_speed = self.max_speed
+        self.requests_per_ten = self.urls_per_tenm
+        upload_name = self.phone_data[-1].split('.')[-1]
+        self.created_cx = self.http_profile.created_cx = self.convert_to_dict(self.phone_data)
         if self.preCleanUp:
-            self.http_profile.created_cx=self.convert_to_dict(self.phone_data)
+            # self.created_cx=self.http_profile.created_cx=self.convert_to_dict(self.phone_data)
             self.precleanup()
         logging.info("Creating Layer-4 endpoints from the user inputs as test parameters")
         time.sleep(5)
         self.http_profile.created_cx.clear()
-        self.http_profile.create(ports=self.phone_data, sleep_time=.5,upload_name=upload_name,
-                                 suppress_related_commands_=None, http=True,
-                                 http_ip=self.url, interop=True, proxy_auth_type=74240,media_source=self.media_source,media_quality=self.media_quality,timeout=1000)
-        
+        # self.create(ports=self.phone_data, sleep_time=.5,upload_name=upload_name,
```

Comment:
If this function is no longer used/present, good to delete. No change required, though